prowler/providers/kubernetes/kubernetes_provider.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+
+from kubernetes import client, config
+
+from prowler.lib.logger import logger
+
+
+class Kubernetes_Provider:
+    def __init__(
+        self,
+        kubeconfig_file: str,
+        context: list,
+    ):
+        logger.info("Instantiating Kubernetes Provider ...")
+        self.api_client, self.context = self.__set_credentials__(
```

Comment:
What is the difference between the input `context: list` and the returned `self.context`?

Reply:
the input context is a str (I will change it now) of the user-defined context that the user wants to scan, whereas the returned self.context is a dict with the context as a key and more information about it retrieved from the kubernetes library as the value.